docs/excel/custom-functions-overview.md

```diff
@@ -17,7 +17,7 @@ function ADD42(a, b) {
 Custom functions are now available in Developer Preview on Windows, Mac, and Excel Online. Follow these steps to try them:
 
 1.  Install Office (build 9325 on Windows or 13.329 on Mac) and join the [Office Insider](https://products.office.com/office-insider) program. (Note that it isn't enough just to get the latest build; the feature will be disabled on any build until you join the Insider program)
-2.  Clone the [Excel-Custom-Functions](https://github.com/OfficeDev/Excel-Custom-Functions) repo and follow the instructions in the README.md to start the add-in in Excel, make changes in the code, and debug.
+2.  Create an Excel-Custom-Functions using Yo Office (https://github.com/OfficeDev/generator-office), and follow the instructions in the project README.md to start the add-in in Excel, make changes in the code, and debug.
```

Comment:
The instructions on the generator-office repo readme don't really go into the details you're mentioning. Maybe I'm missing something, but they don't seem to discuss starting the add-in in Excel.

Reply:
They do mention the "Excel Custom Functions (Preview: Requires the Insider channel for Excel)" project type. If users install and run Yo Office, the first prompt they will see is what project type they want to create. Choosing Excel Custom Functions (Preview: Requires the Insider channel for Excel) will create the custom functions project.  I could call this out more explicitly  in this document if that's the concern

Reply:
I was comparing the yo office readme to the Excel-Custom-Functions one. The ECF readme is very explicit about how to use CFs in Excel, whereas the yo office documentation is general for any usage of yo office.  There's some good content [here](https://github.com/OfficeDev/Excel-Custom-Functions) about debugging and sideloading. Maybe that's obsolete now, though.

Reply:
With this change, we will still be pointing the user to the ECF readme.  Perhaps this is the part that's confusing? " …. and follow the instructions in the project README.md to start the add-in in Excel"  I can see how a user might think this refers to the Yo Office ReadMe and not the ECF ReadMe. If that indeed is the point of confusion I will make this clearer

Reply:
Yeah, that'd be great. Maybe use markdown links for clarity. `[Yo Office] (https://github.com/OfficeDev/generator-office), and follow the instructions in the [project README.md](https://github.com/OfficeDev/Excel-Custom-Functions/blob/master/README.md)`